dash/dash.py

```diff
@@ -638,6 +638,12 @@ def _validate_callback(self, output, inputs, state):
         # pylint: disable=too-many-branches
         layout = self._cached_layout or self._layout_value()
 
+        for i in inputs:
+            if output == i:
```

Comment:
Going to need some changes for multi-output...

Reply:
Yep, I'll update once merged.